_posts/2015/2015-11-16-carbon-automobiles-bebop-fashion.md

```diff
@@ -0,0 +1,148 @@
+
+Carbon and automobiles show what can happen when we create open platforms. What happens when everyone then builds on these open platforms?
+
+## Bebop
+
+Bebop is a form of Jazz that started to make an appearance in the 1940s, fifty years into the evolution of jazz.
+
+Like carbon and automobiles, Jazz is also a stacked platform, built on African rhythms, European harmonic accompaniment, the blue notes of racial tension, and a surplus of post-civil-war military instruments. Jazz is known for being incredibly competitive. In Jam sessions musicians take turns improvising, trying to one-up each other and then mimicking the sound they just heard.
+
+The evolution of Jazz is marked by the repeated pattern of someone playing in a slightly different style, and then everyone else imitating it. If you were a trumpet player, and you heard Louis Armstrong, then you wanted to play like him, because that's the greatest trumpet you've ever heard.
+
+Charlie Parker was instrumental in the creation of the style of Jazz called Bebop. There’s an anecdote that the reason he played in this style that was because he didn’t think white musicians could copy him. But eventually they did, because Bebop is what most of us think of when we think Jazz.
+
+Charlie’s innovation was created on the open platform of Jazz. When you chose to build on an open platform, you get a leg up. Charlie didn’t have to popularize Jazz or the blues scales or invent the saxophone. His innovation was taking things that already existed and combined them in interesting ways. Bebop may or may not have happened if Charlie Parker wasn’t involved, but once it existed, it wasn’t terribly difficult for other musicians with similar backgrounds to incorporate his innovation.
+
+We are seeing this same thing happen now in software. Even the most innovate of companies, like Apple, are at most a few years ahead of their competition in certain ways.
+
+What happens when openness prevails?
+
+## Fashion
+
+In fashion, not only is copying allowed, it is the norm. Courts, in the US at least, have ruled that apparel is too utilitarian for copyright & patent protection. They didn’t want a handful of designers owning the building blocks of our clothing. So anyone can copy any garment and sell it as their own design.
+
+In her incredible TED talk, Johanna Blakley lays out the argument that a culture of copying in the fashion industry has created an open ecology of creativity where anyone can use the best ideas. Fashion designers have pleasure of sampling from the entire history of fashion design without talking to a lawyer.
+
+And the result is an industry that is extremely vibrant and innovative.
+
+In her talk, she presents this chart, which is the gross sales in industries with high and low intellectual property protection. On the left is the food industry, automobiles, fashion, and furniture, and on the rights is films, books, and music.
```

Comment:
OTOH many (or their lawyers) in fashion would like to be able to sue other designers and are pushing for increased applicability of various forms of IP or creation of new ones. Maybe doesn't fit into your text, but a point may be that for any industry, both the norms and law are always in play, and participants in the industry can do much to choose whether the direction is toward competition in providing consumer value or competition in suing each other.